base/applications/magnify/magnifier.c

```diff
@@ -18,6 +18,8 @@
 #include <winnls.h>
 #include <shellapi.h>
 #include <windowsx.h>
+#include <stdlib.h>
```

Comment:
Is stdlib.h really needed?

Reply:
I think it's needed, because when i try to delete this line, MSVC compiler gives me an error message "atoi undefined".
It seems that the _ttoi function is a marco defined in tchar.h, which will convert _ttoi automatically into atoi or wtoi depends on the environment.